src/FormulateInput.vue

```diff
@@ -343,10 +349,10 @@ export default {
       var classification = this.$formulate.classify(this.type)
       classification = (classification === 'box' && this.options) ? 'group' : classification
       if (classification === 'box' && this.checked) {
-        return this.value || true
-      } else if (has(this.$options.propsData, 'value') && classification !== 'box') {
-        return this.value
-      } else if (has(this.$options.propsData, 'formulateValue')) {
+        return this.modelValue || true
+      } else if (has(this.$props, 'modelValue') && classification !== 'box') {
```

Comment:
`this.$options.propsData` is no longer a thing. Now there's `this.$props`, but I don't think it behaves in the same way - as I stated in a question on the Vue discord:

> I'm trying to determine whether props were passed to a child component.
> 
> Let's say I have 2 props: 
> ```js
> props: {
>   foo: {
>     type: String,
>     default: '', 
>   },
>   bar: {
>     type: String,
>     default: '', 
>   }
> }
> ```
> And in a consuming component only one prop is passed in:
> ```html
> <my-component foo="someValue" />
> ```
> In Vue 2, I could determine which props are passed in with `.$options.propsData` which would give an object containing only `{ foo: 'someValue'`. However, I'm not sure how to access this same information in Vue 3. I know I have `.$props`, but it seems that also contains `bar` with its default value.
> 
> This means I'm unable to differentiate between a consuming component explicitly passing `bar=""` and `bar`  not being passed at all.
> 
> This is causing some difficulties in helping to port a library from Vue 2 to Vue 3. Any help would be much appreciated!

To which I received the following [response](https://discord.com/channels/325477692906536972/568037134968160256/737438712484593675) from somebody:

> @axwdev I'd consider this a pretty big anti-pattern, and I'd very much recommend you find a different way of achieving what you want.
However, you can still get this data.
If you can find the component vnode, it's present in the props property there.
You can access the vnode from this.$.vnode or getCurrentInstance().vnode.

Reply:
Yeah in my experience there are a lot of things you have to do as a library author that are not exactly accepted common practice. This is one of them for sure.

Reply:
Yeah, the wider the use cases you have to support the more likely you're going have to do unconventional things to accommodate them all!